glean-core/src/util.rs

```diff
@@ -111,6 +111,52 @@ pub(crate) fn truncate_string_at_boundary_with_error<S: Into<String>>(
     }
 }
 
+#[cfg(all(target_arch = "x86", target_os = "windows"))]
+pub mod floating_point_context {
```

Comment:
Any documentation on what this does and why we need it?

Reply:
See https://bugzilla.mozilla.org/show_bug.cgi?id=1623335.  Should probably add a comment with a link to the bug.